gslib/tests/test_cp.py

```diff
@@ -1153,14 +1153,21 @@ def test_other_headers(self):
 
     stdout = self.RunGsUtil(['ls', '-L', dst_uri], return_stdout=True)
     self.assertRegex(stdout, r'Cache-Control\s*:\s*public,max-age=12')
-    self.assertRegex(stdout, r'Metadata:\s*1:\s*abcd')
+
+    if self._use_gcloud_storage:
+      self.assertRegex(stdout, r'"key":\s* "1",\n\s*"value":\s*"abcd"')
```

Comment:
This part may not be necessary because the output of `ls -L` should match now.

Reply:
Tried removing it and got the following error. It looks like `ls -L` is still using the old style. Do you know off hand if there's anything extra I need to enable to get gsutil style formatting? Just pulled from master and got gcloud at head.

AssertionError: Regex didn't match: 'Metadata:\\s*1:\\s*abcd' not found in '
gs://gsutil-test-test-other-headers-bucket-552b913a/foo:
  Creation time:               Fri, 05 Aug 2022 15:25:23 GMT
  Update time:                 Fri, 05 Aug 2022 15:25:23 GMT
  Storage class update time:   Fri, 05 Aug 2022 15:25:23 GMT
  Storage class:               STANDARD
  Cache-Control:               public,max-age=12
  Content-Language:            en
  Content-Length:              810
  Content-Type:                image/gif
  Metadata:
  [
    {
      "key": "1",
      "value": "abcd"
    }
  ]

Reply:
I think you need both the latest gcloud and the latest gsutil commit

Reply:
Seems to work now!